dronekit/__init__.py

```diff
@@ -889,6 +954,14 @@ def listener(self, name, m):
             self._mount_yaw = m.pointing_c / 100
             self.notify_attribute_listeners('mount', self.mount_status)
 
+        self._capabilities = 0
+        self._raw_version = 0
+
+        @self.on_message('AUTOPILOT_VERSION')
```

Comment:
Having this here works, but it is cleaner to include the `on_message` in the class that gets populated IFF this is likely to be the only listener for the message.